Source/Core/VideoBackends/OGL/main.cpp

```diff
@@ -159,7 +159,8 @@ bool VideoBackend::FillBackendInfo()
   g_Config.backend_info.MaxTextureSize = static_cast<u32>(max_texture_size);
   if (max_texture_size < 1024)
   {
-    PanicAlert("GL_MAX_TEXTURE_SIZE too small at %i - must be at least 1024.", max_texture_size);
+    PanicAlertFmtT("GL_MAX_TEXTURE_SIZE too small at {0} - must be at least 1024.",
+                   max_texture_size);
```

Comment:
I think translators are likely to misinterpret the word "at" in this string. What do you think about something like "GL_MAX_TEXTURE_SIZE is {0} - must be at least 1024"